docs/assets/api-gateway-request-flow.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is veeery much outdated. We should either update that flow, or get rid of that entirely

Reply:
I created an issue for updating it: https://github.com/kyma-project/api-gateway/issues/549